### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by mina-andajos**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
 <!-- 
 Thanks for your contribution!
+Please fill out the neccessary sections, and delete unused ones.
 -->
+Fixes #issue-number
+----------
+## Type of change:
+- [ ] Bug fix (non-breaking change which fixes an issue).
+- [ ] New feature (non-breaking change which adds functionality).
```
Comment (tristanlatr) [inline .github/pull_request_template.md:9]: There is not concept of breaking change in saskatoon right now. So I don;t think we should add this stuff in perenthesis.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
```
Comment (tristanlatr) [inline .github/pull_request_template.md:10]:
```suggestion
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,3 +1,35 @@
...
+
+-------
+## Checklist:
+- [ ] My code follows the [style guidelines](https://github.com/LesFruitsDefendus/saskatoon-ng/blob/develop/CONTRIBUTING.md) of this project.
+- [ ] I have performed a self-review of my own code.
+- [ ] I have commented my code, particularly in hard-to-understand areas.
+- [ ] I have made corresponding changes to the documentation.
+- [ ] My changes generate no new warnings.
+- [ ] New and existing unit tests pass locally with my changes.
+- [ ] Any dependent changes have been merged and published in downstream modules.
```
Comment (tristanlatr) [inline .github/pull_request_template.md:35]: I don't think this is relevant for us
```suggestion

```

**Review by tristanlatr — CHANGES_REQUESTED** [review]
Hi @mina-andajos,

Thanks a lot, it looks great.

File: .github/pull_request_template.md (hunk 1 above)
Comment (mina-andajos) [reply .github/pull_request_template.md:9]: Wouldn't for example dependency changes or some major changes break the environment for some people?
I could maybe add this instead:

```markdown
- [ ] Change to the app models (requires making migrations).
```
What do you think? @tristanlatr

Comment (tristanlatr) [reply .github/pull_request_template.md:9]: Yes that sounds good

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -1,3 +1,34 @@
 <!-- 
 Thanks for your contribution!
+Please fill out the neccessary sections, and delete unused ones.
 -->
+Fixes #issue-number
+----------
+## Type of change:
+- [ ] Bug fix (non-breaking change which fixes an issue).
+- [ ] New feature (non-breaking change which adds functionality).
```
Comment (tristanlatr) [inline .github/pull_request_template.md:9]:
```suggestion
- [ ] New feature (change which adds functionality).
```

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -1,3 +1,34 @@
 <!-- 
 Thanks for your contribution!
+Please fill out the neccessary sections, and delete unused ones.
 -->
+Fixes #issue-number
+----------
+## Type of change:
+- [ ] Bug fix (non-breaking change which fixes an issue).
```
Comment (tristanlatr) [inline .github/pull_request_template.md:8]:
```suggestion
- [ ] Bug fix (change which fixes an issue).
```

**Review by tristanlatr — CHANGES_REQUESTED** [review]
Looks good, only that breaking change is not a thing for us at the moment, so I'de suggest simply not use that term since we can't define it correctly.

**Review by tristanlatr — APPROVED** [review]
Thanks @mina-andajos

**Pull request merged**